heartbeat/docs/reference/configuration/heartbeat-options.asciidoc

```diff
@@ -0,0 +1,448 @@
+specified, Heartbeat establishes a TCP connection even if the monitor is
+configured to use SSL. If `tls` or `ssl` is specified, Heartbeat establishes
+an SSL connection. However, if the monitor is not configured to use SSL, the
+system defaults are used (not supported on Windows).
+** `host` is the hostname.
+** `port` is the port number. If `port` is missing in the URL, the
+<<monitor-tcp-ports,`ports`>> setting is required.
+
+// Questions: Can port above be a list of ports, or is it only a single port.
+
+[[monitor-tcp-ports]]
+===== ports
+
+A list of ports to ping if the host specified in <<monitor-tcp-hosts,`hosts`>>
+does not contain a port number.
+
+Example configuration:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+- type: tcp 
+  schedule: '@every 5s'
+  hosts: ["myhost"]
+  ports: [80, 9200, 5044]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+-------------------------------------------------------------------------------
+
+[[monitor-tcp-check]]
+===== check
+
+An optional payload string to send to the remote host and the expected answer.
+If no payload is specified, the endpoint is assumed to be available if the
+connection attempt was successful. If `send` is specified without `receive`,
+any response is accepted as OK. If `receive` is specified without `send`, no
+payload is sent, but the client expects to receive a payload on connect.
+
+Example configuration:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+- type: tcp 
+  schedule: '@every 5s'
+  hosts: ["myhost"]
+  ports: [80, 9200, 5044]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+  check:
+    send: ''
+    receive: ''
+-------------------------------------------------------------------------------
+
+
+//Steffen: Can you provide a good example of the values to set for send and receive in the above example?
+
+[[monitor-tcp-proxy-url]]
+===== proxy_url
+
+The SOCKS5 proxy URL.
+
+//Steffen: Can we say more about this option?
+
+[[monitor-tcp-proxy-use-local-resolver]]
+===== proxy_use_local_resolver
+
+A Boolean value that specifies whether to resolve hostnames locally instead of
+resolving them on the SOCKS5 server. The default is false.
+
+[[monitor-tcp-tls-ssl]]
+===== ssl
+
+The TLS/SSL connection settings. 
+
+//Steffen: Is this option so that you can check to see if the servers certificates are valid?
+
+Example configuration:
+
+[source,yaml]
+-------------------------------------------------------------------------------
+- type: tcp 
+  schedule: '@every 5s'
+  hosts: ["myhost"]
+  ports: [80, 9200, 5044]
+  ipv4: true
+  ipv6: true
+  mode: any
+  timeout: 16s
+  ssl:
+    certificate_authorities: ['/etc/ca.crt']
+    supported_protocols: ["TLSv1.0", "TLSv1.1", "TLSv1.2"]
+-------------------------------------------------------------------------------
+
+
+//Steffen: How are the values for supported_protocols used?
```

Comment:
it's this one: https://github.com/elastic/beats/blob/master/libbeat/docs/outputconfig.asciidoc#versions

Makes me wonder which one is correct. Docs or sample config? Found it, doc says `version, but correct is [supported_protocols](https://github.com/elastic/beats/blob/master/libbeat/outputs/tls.go#L33)